setup.py

```diff
@@ -19,17 +19,17 @@
 
 def readme():
     """Read in and return the contents of the project's README.md file."""
-    with open("README.md", encoding="utf-8") as f:
-        return f.read()
+    with open("README.md", encoding="utf-8") as in_file:
+        return in_file.read()
 
 
 # Below two methods were pulled from:
 # https://packaging.python.org/guides/single-sourcing-package-version/
 def read(rel_path):
     """Open a file for reading from a given relative path."""
     here = abspath(dirname(__file__))
-    with codecs.open(join(here, rel_path), "r") as fp:
-        return fp.read()
+    with codecs.open(join(here, rel_path), "r") as in_file:
+        return in_file.read()
```

Comment:
This change from the skeleton is unnecessary and should be reverted:
```suggestion
    with codecs.open(join(here, rel_path), "r") as fp:
        return fp.read()
```